src/core/constants.js

```diff
@@ -22,8 +22,18 @@ export const VERSION =
  */
 export const P2D = 'p2d';
 /**
- * One of the two render modes in p5.js: P2D (default renderer) and WEBGL
- * Enables 3D render by introducing the third dimension: Z
+ * One of the two render modes in p5.js, most known for 3D rendering.
+ *
+ * Some differences and new features to expect compared to the default render mode <a href="/#/p5/P2D">P2D</a>:
+ * - The <strong>coordinate system</strong> is centered and uses an optional z-dimension
+ * - <strong>Textures</strong>, <strong>materials</strong>, and <strong>lighting</strong> can be applied to shapes
+ * - You can control the <strong>camera</strong> that views the scene
+ * - <strong>Text</strong> is drawn differently
+ * - You can use <strong>shaders</strong>
```

Comment:
I think we can also add a couple more elements to this list.
**3D Shapes** - WEBGL mode can be used to draw 3D shapes like cubes, spheres, cones, and more. (with links to each shape). 

**Graphics Acceleration** - WEBGL mode uses the graphics card and may help you boost performance of your sketch or draw an increased number of shapes to the screen at once.

Any other major differences worth mentioning here?